boards/FSM/Src/main.c

```diff
@@ -148,6 +149,16 @@ static void CanTxQueueOverflowCallBack(size_t overflow_count)
     App_CanTx_SetPeriodicSignal_TX_OVERFLOW_COUNT(can_tx, overflow_count);
 }
 
+static bool ReturnTrue(void)
+{
+    return true;
+}
+
+static void DemoCallBack(void)
+{
+    printf("Test\n");
+}
```

Comment:
I'm assuming we want to remove the changes to this file before we merge?

Reply:
Yup!

Reply:
I'm hoping to write a real-example before this gets merged in, but for now I will just stick with these dummy functions so I can get things to compile